lexibank_yanglalo.py

```diff
@@ -67,7 +67,7 @@ def cmd_install(self, **kw):
 
                         for row in ds.add_lexemes(
                             Language_ID=language,
-                            Parameter_ID=parameter,
+                            Parameter_ID=concept_map[parameter],
                             Form=form,
                             Value=value,
                             Segments=segments,
```

Comment:
Segments should be derived from the orthography profile, right? Could you please also check that this works and remove the explicit `self.tokenizer` line, as well as the explicit `form = `? This is one of the crucial check points as well. Let me know if this is unclear.

Reply:
Sure, will do it.